src/patternfly/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,158 @@
+$pf-c-dropdown__toggle--PaddingRight: $pf-global--spacer--md !default;
+$pf-c-dropdown__toggle--PaddingBottom: $pf-global--spacer--xs !default;
+$pf-c-dropdown__toggle--PaddingLeft: $pf-global--spacer--sm !default;
+$pf-c-dropdown__toggle--FontSize: $pf-global--FontSize--md !default;
+$pf-c-dropdown__toggle--BackgroundColor: $pf-global--BackgroundColor--light-100 !default;
+$pf-c-dropdown__toggle--BorderWidth: $pf-global--BorderWidth--sm !default;
+$pf-c-dropdown__toggle--BorderColor: $pf-global--Color--dark-200 !default;
+$pf-c-dropdown__toggle--Color: $pf-global--Color--dark-100 !default;
+
+$pf-c-dropdown__toggle--hover--BorderWidth: $pf-global--BorderWidth--md !default;
+
+$pf-c-dropdown__toggle--is-expanded--BorderColor: $pf-global--Color--active !default;
+$pf-c-dropdown__toggle--is-expanded--BorderWidth: $pf-global--BorderWidth--md !default;
+
+// Toggle Arrow
+$pf-c-dropdown__arrow--MarginLeft: $pf-global--spacer--md !default;
+$pf-c-dropdown__arrow--BorderWidth: $pf-global--BorderWidth--md !default;
+
+// Menu
+$pf-c-dropdown__menu--BackgroundColor: $pf-global--BackgroundColor--light-200 !default;
+$pf-c-dropdown__menu--BoxShadow: $pf-global--BoxShadow--sm !default;
+
+// Menu Item
+$pf-c-dropdown__menu-item--PaddingTop: $pf-global--spacer--sm !default;
+$pf-c-dropdown__menu-item--PaddingRight: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--PaddingBottom: $pf-global--spacer--sm !default;
+$pf-c-dropdown__menu-item--PaddingLeft: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--first--PaddingTop: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--last--PaddingBottom: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--Color: $pf-global--Color--dark-100 !default;
+$pf-c-dropdown__menu-item--is-disabled--Color: $pf-global--Color--dark-200 !default;
+
+$pf-c-dropdown__menu-item--hover--BackgroundColor: $pf-global--BackgroundColor--light-300 !default;
+
+// Menu Item Separator
+$pf-c-dropdown__separator--Height: $pf-global--BorderWidth--md !default;
+$pf-c-dropdown__separator--BackgroundColor: $pf-global--BackgroundColor--light-300 !default;
+
+
+.pf-c-dropdown {
+  position: relative;
+  display: inline-block;
+
+  &__toggle {
+    padding: $pf-c-dropdown__toggle--PaddingTop $pf-c-dropdown__toggle--PaddingRight $pf-c-dropdown__toggle--PaddingBottom $pf-c-dropdown__toggle--PaddingLeft;
+    font-size: $pf-c-dropdown__toggle--FontSize;
+    color: $pf-c-dropdown__toggle--Color;
+    background-color: $pf-c-dropdown__toggle--BackgroundColor;
+    border: $pf-c-dropdown__toggle--BorderWidth solid $pf-c-dropdown__toggle--BorderColor;
+
+    &:hover {
+      padding-top: calc(#{$pf-c-dropdown__toggle--PaddingTop} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-right: calc(#{$pf-c-dropdown__toggle--PaddingRight} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
```

Comment:
@srambach @mattnolting Are we using an inner shadow instead of a calculation on the padding to show active states on btns, dropdown and inputs?